src/DependencyInjection/PsyshExtension.php

```diff
@@ -50,16 +51,21 @@ public function load(array $configs, ContainerBuilder $container): void
 
         $config = $this->processConfiguration(new Configuration(), $configs);
 
+        $containerId = 'test.service_container';
+
         foreach ($config['variables'] as $name => $value) {
-            if (is_string($value) && strpos($value, '@') === 0) {
-                $value = new Reference(substr($value, 1));
+            if (is_string($value)) {
+                if (strpos($value, '@') === 0) {
+                    $value = new Reference(substr($value, 1));
+                } else if (strpos($value, '{') === 0) {
```

Comment:
As mentioned above I think if we are to introduce support for expressions we should we the "standard" expression? Or maybe I'm confusing with something else... Maybe OverblogGraphQLBundle YAML config